downstream/titles/release-notes/topics/installer-24-3.adoc

```diff
@@ -0,0 +1,39 @@
+// This is the release notes file for AAP 2.4 async installer release 2.4-3 dated November 21, 2023
+
+[id="installer-24-3"]
+
+= RHBA-2023:7460 - installer release 2.4-3 - November 21, 2023
+
+link:https://access.redhat.com/errata/RHBA-2023:7460[RHBA-2023:7460]
+
+== General
+
+* Fixed an error which caused the incorrect target database to be selected when restoring {EDAName} from a backup (AAP-18151)
+
+* Postgres tasks which create users in FIPS environments now use `scram-sha-256` (AAP-17516)
+
+* All {EDAName} services are enabled after installation is complete (AAP-17426)
+
+* Ensure all backup/restore staged files and directories are cleaned up before running backup/restore and marked for deletion after backup/restore (AAP-16101)
+
+* Updated nginx to 1.22 (AAP-15962)
+
+* Added a task to VMs that will run the `awx-manage` command to pre-create events table partition(s) before executing `pg_dump` and added a variable for the default number of hours to pre-create (AAP-15920)
```

Comment:
Always default to plural instead of using "(s)", even if it could be singular as well as plural.  

https://www.ibm.com/docs/en/ibm-style?topic=grammar-plurals#-s-

Reply:
Noted 👍